### DISCUSSION TITLE
Use Fly Statics for static assets

### DISCUSSION THREAD

**Discussion by kentcdodds** [epicweb-dev/epic-stack#284] [Ideas]
The web server can handle the static files well without issue, but it would be nice to use Fly's static assets to free up traffic on our web server: https://fly.io/docs/reference/configuration/#the-statics-sections

Unfortunately this is on hold for now due to the caveat that you cannot specify cache headers so the browser will have to re-request these assets every time which will lead to worse perf in practice.

Having our web server handle these is fine, but we'll want to keep an eye on this feature for when it does allow specifying custom cache-control headers.

**Comment by weleo** [comment]
What do you think about using [Tigris](https://fly.io/docs/reference/tigris/) for static assets compared to just using the web server?
